pymc/distributions/truncated.py

```diff
@@ -290,7 +290,9 @@ class Truncated(Distribution):
         with pm.Model():
             normal_dist = pm.Normal.dist(mu=0.0, sigma=1.0)
             truncated_normal = pm.Truncated("truncated_normal", normal_dist, lower=-1, upper=1)
-
+        with pm.Model():
+            normal_dist = pm.Normal.dist(mu=0.0, sigma=1.0)
+            partially_truncated_normal = pm.Truncated("partially_truncated_normal", normal_dist, lower=-np.inf, upper=1)
```

Comment:
Let's put it a s a separate example for more readability with a header explaning it. And include multiple variates like `lower=[-np.inf, -np.inf, 0, 0], upper=[np.inf, 1, np.inf, 1], shape=(4,))` explaining that the first one is not truncated / censored, the second one only from above, the third one, only from below, and the last one from both ends.